src/test/spec/runner/operation.rs

```diff
@@ -261,8 +271,11 @@ impl TestOperation for InsertMany {
         let result = collection
             .insert_many(self.documents.clone(), self.options.clone())
             .await?;
-        let result = bson::to_bson(&result)?;
-        Ok(Some(result))
+        let mut ids = Document::new();
+        for (id, bson) in result.inserted_ids {
+            ids.insert(id.to_string(), bson);
+        }
```

Comment:
This is where I ran into the issue of deserializing an unsigned integer -- creating the document manually as a workaround fixed the problem

Reply:
Hmm, the ids being used are unsigned integers?

Reply:
Oh, I see; the ids are signed, but the keys int the map are `usize`, so they can't be serialized. Another workaround would be to map the keys, e.g. `result.inserted_ids.into_iter().map(|(k, v)|, (k.to_string(), v)).collect()`.